tests/test_compiler.py

```diff
@@ -357,3 +361,9 @@ def test_get_compiler_settings(compiler, project):
 
 def test_evm_version(compiler):
     assert compiler.config.evm_version == "constantinople"
+
+
+def test_source_map(project, compiler):
+    source_path = project.contracts_folder / "MultipleDefinitions.sol"
+    result = compiler.compile([source_path])[-1]
+    assert result.sourcemap.__root__ == "124:87:0:-:0;;;;;;;;;;;;;;;;;;;"
```

Comment:
Can make this test nicer if https://github.com/ApeWorX/ethpm-types/pull/58 merge + releases